strings/mother.json

```
@@ -1,49 +1,49 @@
 {
 	"do_something": [
-		"CLEAN YOUR ROOM THIS INSTANT!",
-		"DON'T SIT THAT CLOSE TO THE TV!",
-		"FOR GOD'S SAKE, GO TAKE A SHOWER!!",
-		"IT'S TIME TO WAKE UP FOR SCHOOL!!",
-		"PAUSE THAT ONLINE GAME! NOW!",
-		"PUT SOME CLOTHES ON! YOU'LL CATCH A COLD!",
-		"STOP ASKING FOR MONEY, I'M NOT AN ATM!",
-		"WATCH YOUR MOUTH, CHILD!!",
-		"WHY DON'T YOU ANSWER MY PHONE CALLS?!",
-		"YOU SHOULD @pick(verb) YOUR @pick(relative) ONCE IN A WHILE!"
+		"УБЕРИСЬ В КОМНАТЕ СЕЙЧАС ЖЕ!",
+		"НЕ СИДИ ТАК БЛИЗКО К ТЕЛЕВИЗОРУ!",
+		"РАДИ ВСЕГО СВЯТОГО, ПРИМИ ДУШ!!",
+		"ПОДЪЁМ, ПОРА В ШКОЛУ!!",
+		"ПОСТАВЬ ЭТУ ОНЛАЙН-ИГРУ НА ПАУЗУ! СЕЙЧАС ЖЕ!",
+		"НАДЕНЬ ЧТО-НИБУДЬ! ПРОСТУДИШЬСЯ!",
+		"ПЕРЕСТАНЬ ПРОСИТЬ ДЕНЬГИ, Я НЕ БАНКОМАТ!",
+		"СЛЕДИ ЗА СЛОВАМИ, ПИЗДЮК!!",
+		"ПОЧЕМУ ТЫ НЕ БЕРЁШЬ ТРУБКУ КОГДА Я ЗВОНЮ?!",
+		"ПОЧЕМУ ТЫ НЕ @pick(verb) @pick(relative)?! СОВСЕМ ЗАБЫЛ ПРО ВСЕХ! ТОЛЬКО В СВОИ ИГРУШКИ И ИГРАЕШЬ!"
 	],
 
 	"be_upset": [
-		"BECAUSE I SAID SO!",
-		"I DON'T CARE WHAT YOU SAY!",
-		"I'M NOT ASKING; I'M TELLING!!",
-		"I WASN'T BORN YESTERDAY!",
-		"MONEY DOESN'T GROW ON TREES!",
-		"WHAT DID I DO TO DESERVE A KID LIKE THIS...",
-		"USELESS!",
-		"YOU INSULT YOUR GRANDPARENTS!"
+		"ПОТОМУ ЧТО Я ТАК СКАЗАЛА!",
+		"МНЕ ВСЁ РАВНО, ЧТО ТЫ ГОВОРИШЬ!",
+		"Я НЕ ПРОШУ, Я ГОВОРЮ!!",
+		"Я НЕ ВЧЕРА РОДИЛАСЬ!",
+		"ДЕНЬГИ НА ДЕРЕВЬЯХ НЕ РАСТУТ!",
+		"ЧТО Я СДЕЛАЛА, ЧТОБЫ ЗАСЛУЖИТЬ ТАКОГО РЕБЁНКА...",
+		"БЕСПОЛЕЗНЫЙ!",
+		"ТЫ ОСКОРБЛЯЕШЬ СВОИХ БАБУШКУ И ДЕДУШКУ!"
 	],
 
 	"get_reprimanded": [
-		"I BROUGHT YOU INTO THIS WORLD, I CAN TAKE YOU OUT!!!",
-		"I'M GOING TO THROW A FLIP-FLOP AT YOU!!",
-		"NO VIDEOGAMES FOR THE REST OF THE DAY!",
-		"WAIT UNTIL YOUR FATHER GETS HOME!",
-		"YOU'LL THANK ME ONE DAY!",
-		"YOU'RE DISOWNED!!!",
-		"YOU'RE GROUNDED!!"
+		"Я ТЕБЯ ПОРОДИЛА, Я ЖЕ МОГУ ТЕБЯ И УНИЧТОЖИТЬ!!!",
+		"Я СЕЙЧАС ШВЫРНУ В ТЕБЯ ТАПКОМ!!",
+		"НИКАКИХ ИГР ДО КОНЦА ДНЯ!",
+		"ПОДОЖДИ, КОГДА ПРИДЁТ ТВОЙ ОТЕЦ!",
+		"ТЫ МНЕ ЕЩЁ СПАСИБО СКАЖЕШЬ!",
+		"ТЫ ЛИШЁН НАСЛЕДСТВА!!!",
+		"ТЫ ПОД ДОМАШНИМ АРЕСТОМ!!"
 	],
 
 	"verb": [
-		"CALL",
-		"HELP",
-		"VISIT"
+		"ЗВОНИШЬ",
+		"ПОМОГАЕШЬ",
+		"НАВЕЩАЕШЬ"
```

увы нет способов перевода лучше в данный момент, поэтому так
```suggestion
		"НЕ ХОДИШЬ К"
```